pkg/formula/creator/creator_test.go

```diff
@@ -204,9 +209,21 @@ func TestCreator(t *testing.T) {
 			creator := NewCreator(treeMan, tt.in.dir, tt.in.file, tt.in.tplM)
 			out := tt.out
 			got := creator.Create(in.formCreate)
-			if (got != nil && out.err == nil) || got != nil && got.Error() != out.err.Error() || out.err != nil && got == nil {
+			if (got != nil && out.err == nil) ||
+				got != nil && got.Error() != out.err.Error() ||
+				out.err != nil && got == nil {
 				t.Errorf("Create(%s) got %v, want %v", tt.name, got, out.err)
 			}
```

Comment:
How about making testify standard? Also, this boolean logic seems complex and hard to understand
```suggestion
			if (out.err == nil) {
			      require.NoError(t, got)
			} else {
			     require.NotNil(t, out.err)
			     require.Equal(t, err.Error(), out.err.Error())
			}
```

Reply:
sure! removed